### PR TITLE
Test/fix failing tests after styling changes

### PR DESCRIPTION
**Pull request by lilianaweimer**
@letakeane @khalidwilliams

## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Had some unit tests, but many were failing because of change we made while styling yesterday

## What is the new behavior?

- All unit tests are passing and I added a few

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

It seems like the rests of the remaining things to test will require integration, async, and/or thunks testing so I am going to wait to do those so we can work through them together. There is also an error coming from the `<Lives>` component (the usual `Each child in a list should have a unique "key" prop.` error). I tried to fix it, but it didn't work (not sure why, we can probably figure that out together).

### CODE REVIEW

File: src/Answers/Answers.test.js [hunk 1]
```diff
@@ -72,5 +74,23 @@ describe('Answers', () => {
     expect(answer2).toBeInTheDocument()
     expect(answer3).toBeInTheDocument()
     expect(answer4).toBeInTheDocument()
-  })
+  });
+
+  it('should check if the answer is correct when an answer is clicked', () => {
```
Comment (Billwilke42) [inline src/Answers/Answers.test.js:79]: Thanks for fixing these

File: src/Form/Form.test.js [hunk 2]
```diff
@@ -96,4 +99,22 @@ describe('Form', () => {
...
 
+  it('should fire the handleSubmit function when the button is clicked', () => {
+    const { getByRole } = render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Form />
+        </Provider>
+      </BrowserRouter>);
+      
+    //I think this test requires thunk testing
```
Comment (khalidwilliams) [inline src/Form/Form.test.js:110]: You might be able to just provide your test store a think middleware, then await the assertion on line 116. I'm not sure, could be worth a try, though.

File: src/Header/Header.test.js [hunk 3]
```diff
@@ -27,19 +27,17 @@ describe('Header', () => {
...
-    const lives = getByText('2');
-    const score = getByText('Lives: Score: 4');
-    //these live in the same element ('Lives:' and 'Score:')
-    //so we need to test for them together
-    //also, the lives live in their own element (the actual #)
-    //so they need to be tested for separately 
+    const round = getByText('Round 4: Question 2');
+    const category = getByText('test category');
+    const lives = getByText('Lives:');
+    const score = getByText('Score:');
```
Comment (khalidwilliams) [inline src/Header/Header.test.js:33]: Do these not live in the same element anymore?

File: src/Lives/Lives.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const Lives = (props) => {
   }
   return (
     <p>{lives.map(heart => {
-      return <i className='nes-icon heart is-medium' />
+      return <i className='nes-icon heart is-medium' data-testid='heart'/>
```
Comment (khalidwilliams) [inline src/Lives/Lives.js:11]: Does the `heart` class provide these `i` tags their styling?

File: src/Lives/Lives.test.js [hunk 5]
```diff
@@ -11,10 +11,10 @@ import { createStore } from 'redux';
...
 
-describe('Header', () => {
+describe('Lives', () => {
 
 	it('should render without crashing', () => {
-		const { getByText } = render(
+		const { getByText, getAllByTestId } = render(
 				<BrowserRouter>
 					<Provider store={store}>
             <Header
```
Comment (khalidwilliams) [inline src/Lives/Lives.test.js:20]: Why is the Header still being rendered here? For a unit test like this, you want to make sure that the component under test can be successfully rendered in isolation. It looks like you could just render `<Lives lives={2}/>` without the `Header`, and still make the same assertions ( maybe without lines 30 and 33).

File: src/Answers/Answers.js [hunk 6]
```diff
@@ -11,7 +11,7 @@ const Answers = (props) => {
   const mappedAnswers = shuffledAnswers.map((answer, i) => {
     return (
       <label key={i} id={answer}>
-        <input type='radio' className='nes-radio is-dark' checked/>
+        <input type='radio' className='nes-radio is-dark' checked readOnly/>
       <span 
         key={i} 
         id={answer}
```
Comment (KyleWong2510) [inline src/Answers/Answers.js:17]: What is the readOnly attribute?

File: src/Form/Form.test.js [hunk 7]
```diff
@@ -52,7 +55,7 @@ describe('Form', () => {
   });
 
   it('should be able to fill the form out', () => {
-    const { getByTestId, getByPlaceholderText, getByDisplayValue, debug } = render(
+    const { getByTestId, getByPlaceholderText, getByDisplayValue } = render(
```
Comment (KyleWong2510) [inline src/Form/Form.test.js:58]: Good job revising these!

**Pull request merged**